src/Dependencies/Microsoft.DiaSymReader/ISymUnmanagedAsyncMethod.cs

```diff
@@ -12,13 +12,13 @@ namespace Microsoft.DiaSymReader
     public interface ISymUnmanagedAsyncMethod
     {
         [PreserveSig]
-        int IsAsyncMethod(out bool value);
+        int IsAsyncMethod([MarshalAs(UnmanagedType.Bool)]out bool value);
```

Comment:
This is effectively the PIA definition for these interfaces -- this is the definition that one must use if consuming or implementing these interfaces from managed code in Visual Studio.